src/java/za/co/mip/ablduck/models/DataJSObject.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright 2017 MIP Holdings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package za.co.mip.ablduck.models;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.lang.Object;
+
+import com.google.gson.annotations.Expose;
+
+import za.co.mip.ablduck.models.data.ClassDataObject;
+import za.co.mip.ablduck.models.data.MemberTypeDataObject;
+import za.co.mip.ablduck.models.data.SearchDataObject;
+import za.co.mip.ablduck.models.data.SignatureDataObject;
+import za.co.mip.ablduck.models.data.SubsectionDataObject;
+
+public class DataJSObject {
+    //Empty for now
+    @Expose
+    public List<String> guides = new ArrayList<>();
+    @Expose
+    public List<String> videos = new ArrayList<>();
+    @Expose
+    public List<String> examples = new ArrayList<>();
+    @Expose
+    public Object guideSearch = new Object();
+
+    //Implemented
+    @Expose
+    public String localStorageDb = "docs";
+
+    @Expose
+    public String commentsUrl = "";
+
+    @Expose
+    public String commentsDomain = "";
+
+    @Expose
+    public String message = "";
+
+    @Expose
+    public Boolean tests = false;
+
+    @Expose
+    public Boolean showPrintButton = true;
+
+    @Expose
+    public Boolean touchExamplesUi = false; //TODO: Need to remove this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1135)